src/main/java/edu/cornell/kfs/module/purap/jaggaer/supplier/xml/CustomElement.java

```diff
@@ -0,0 +1,86 @@
+
+package edu.cornell.kfs.module.purap.jaggaer.supplier.xml;
+
+import java.util.ArrayList;
+import java.util.List;
+import jakarta.xml.bind.annotation.XmlAccessType;
+import jakarta.xml.bind.annotation.XmlAccessorType;
+import jakarta.xml.bind.annotation.XmlAttribute;
+import jakarta.xml.bind.annotation.XmlElement;
+import jakarta.xml.bind.annotation.XmlElements;
+import jakarta.xml.bind.annotation.XmlRootElement;
+import jakarta.xml.bind.annotation.XmlType;
+import jakarta.xml.bind.annotation.adapters.CollapsedStringAdapter;
+import jakarta.xml.bind.annotation.adapters.NormalizedStringAdapter;
+import jakarta.xml.bind.annotation.adapters.XmlJavaTypeAdapter;
+
+@XmlAccessorType(XmlAccessType.FIELD)
+@XmlType(name = "", propOrder = { "customElementIdentifier", "displayName", "customElementValueListOrAttachments" })
+@XmlRootElement(name = "CustomElement")
+public class CustomElement {
+
+    @XmlAttribute(name = "isChanged")
+    @XmlJavaTypeAdapter(CollapsedStringAdapter.class)
+    protected String isChanged;
+    @XmlAttribute(name = "isActive")
+    @XmlJavaTypeAdapter(CollapsedStringAdapter.class)
+    protected String isActive;
+    @XmlAttribute(name = "type")
+    @XmlJavaTypeAdapter(NormalizedStringAdapter.class)
+    protected String type;
+    @XmlElement(name = "CustomElementIdentifier")
+    protected JaggaerBasicValue customElementIdentifier;
+    @XmlElement(name = "DisplayName")
+    protected DisplayName displayName;
```

Comment:
How is display name different from CustomElementIdentifier?

Reply:
DisplayName is used in multiple locations, so I wanted to keep it distinct